sdk/spring/spring-cloud-azure-service/src/main/java/com/azure/spring/cloud/service/implementation/identity/impl/credential/adapter/CacheableSpringTokenCredential.java

```diff
@@ -0,0 +1,22 @@
+package com.azure.spring.cloud.service.implementation.identity.impl.credential.adapter;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.spring.cloud.service.implementation.identity.api.credential.TokenCredentialProviderOptions;
+
+public class CacheableSpringTokenCredential extends CacheableTokenCredentialAdapter<TokenCredential> {
+
+    public CacheableSpringTokenCredential(TokenCredentialProviderOptions options,
+                                          TokenCredential delegate) {
+        super(options, delegate);
+    }
+
+    @Override
+    protected Descriptor[] getTokenCredentialKeyDescriptors() {
+        return new Descriptor[] {
+                Descriptor.AUTHORITY_HOST,
+                Descriptor.TENANT_ID,
+                Descriptor.CLIENT_ID
+        };
+    }
```

Comment:
Is the only thing matters in `CacheableSpringTokenCredential` the beanName?

Reply:
Sorry, I didn't get your point,

If there is only one springTokenCredential instance, the prefix of cachekey with these descriptors is enough.

Reply:
If the TokenCredential is a Spring-managed TokenCredential, should the descriptor only contain the Spring-managed TokenCredential's bean name?